lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelAnalysis.cxx

```diff
@@ -316,24 +315,34 @@ Scalar LinearModelAnalysis::getFisherPValue() const
   return FisherSnedecor(dofModel, dof).computeComplementaryCDF(FStatistic);
 }
 
-/* Kolmogorov-Smirnov normality test */
+/* Estimator of the standard deviation */
+Scalar LinearModelAnalysis::getResidualsStandardError() const
+{
+  const Scalar sigma2 = linearModelResult_.getSampleResiduals().computeRawMoment(2)[0];
+  const UnsignedInteger dof = linearModelResult_.getDegreesOfFreedom();
+  const UnsignedInteger n = linearModelResult_.getSampleResiduals().getSize();
+  // TODO: The formula sigma2 * sampleSize / dof is highly suspiscious: this is already unbiased in LinearModelAlgorithm
+  const Scalar stdError = std::sqrt(sigma2 * n / dof);
+  return stdError;
+}
+
+/* Kolmogorov-Smirnov normality test of the residuals */
 TestResult LinearModelAnalysis::getNormalityTestResultKolmogorovSmirnov() const
 {
-  // We check that residuals are centered with variance = sigma2
+  // We check that residuals have a Normal distribution
   const Sample residuals(linearModelResult_.getSampleResiduals());
-  // Compute Sigma2
-  const Scalar sigma2(residuals.computeRawMoment(2)[0]);
-  const Normal dist(0.0, std::sqrt(sigma2));
+  const Scalar sigma(getResidualsStandardError());
```

Comment:
If the residuals are Gaussian with zero mean, then the unbiased estimator of the variance is from `sigma2(residuals.computeRawMoment(2)[0])`. Please remove the call to `getResidualsStandardError()`.

Reply:
And use
```
const Scalar sigma = getResidualsStandardError();
```
for simple types

Reply:
Ok : fixed at 38658f2b2